docs/modules/ROOT/pages/community.adoc

```diff
@@ -21,6 +21,9 @@ We welcome your involvement in the Spring Security project.
 There are many ways to contribute, including answering questions on Stack Overflow, writing new code, improving existing code, assisting with documentation, developing samples or tutorials, reporting bugs, or simply making suggestions.
 For more information, see our https://github.com/spring-projects/spring-security/blob/main/CONTRIBUTING.adoc[Contributing] documentation.
 
+NOTE: Since the Stream API is https://github.com/spring-projects/spring-security/issues/7154[much slower] than `for` loop, please use it judiciously.
```

Comment:
Will you please remove it from this document? Since it is in the contributing guidelines already, there's no reason to duplicate.